src/Common/src/Interop/Unix/System.Native/Interop.HostEntry.cs

```diff
@@ -45,9 +45,6 @@ internal unsafe struct HostEntry
         [DllImport(Libraries.SystemNative, EntryPoint = "SystemNative_GetHostEntryForName")]
         internal static extern unsafe int GetHostEntryForName(string address, HostEntry* entry);
 
-        [DllImport(Libraries.SystemNative, EntryPoint = "SystemNative_GetHostByName")]
-        internal static extern unsafe int GetHostByName(string address, HostEntry* entry);
```

Comment:
`SystemNative_GetHostByName` may be removed from

`src/Native/Unix/System.Native/pal_networking.h`
`src/Native/Unix/System.Native/pal_networking.c`

Then the C functions that `SystemNative_GetHostByName` calls in pal_networking.c, that are unusued after the removal may also be removed.

Then some macros like `HAVE_THREAD_SAFE_GETHOSTBYNAME_AND_GETHOSTBYADDR` which will be unused may be removed from `src/Native/Unix/Common/pal_config.h.in` and related lines/functions from CMake file `src/Native/Unix/configure.cmake`.

Reply:
Thank's!